lxd/storage/backend_lxd.go

```diff
@@ -4016,9 +4023,10 @@ func (b *lxdBackend) ActivateBucket(bucketName string, op *operations.Operation)
 		return nil, fmt.Errorf("Remote buckets cannot be activated")
 	}
 
-	vol := b.GetVolume(drivers.VolumeTypeBucket, drivers.ContentTypeFS, bucketName, nil)
+	storageBucketName := project.StorageVolume(projectName, bucketName)
+	storageBucket := b.GetVolume(drivers.VolumeTypeBucket, drivers.ContentTypeFS, storageBucketName, nil)
```

Comment:
Can we leave this as either vol (as it represents a volume that serves the bucket) or bucketVol so it matches the argument name of miniod.EnsureRunning